third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/Lookup.java

```diff
@@ -12,14 +12,17 @@
 
 public class Lookup {
 
-    private Map<String, String> globalLookup = new HashMap<>();
-    private Map<String, Map<String, String>> localLookupByFileName = new HashMap<>();
+    private static final int INITIAL_CAPACITY = 10000;

```

Comment:
Set large Map capacity to *try* and reduce the number of copy/ increases needed.